check-yum-updates/check-yum-updates.sh

```diff
@@ -39,8 +39,13 @@ elif grep -q -i "release 7" /etc/redhat-release ; then
   IMPORTANT=$(yum updateinfo list --sec-severity=Important | grep Important | wc -l)
   LOW=$(yum updateinfo list --sec-severity=Low | grep Low | wc -l)
   CRITICAL=$(yum updateinfo list --sec-severity=Critical | grep Critical | wc -l)
+elif grep -q -i "release 8" /etc/redhat-release ; then
+  MODERATE=$(yum updateinfo list --sec-severity=Moderate | grep Moderate | wc -l)
```

Comment:
This should rather be `dnf` (although `yum` exists as a symlink to `dnf`, so it will work as it is).